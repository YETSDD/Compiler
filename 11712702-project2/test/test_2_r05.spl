float add(int a,int b){
    return 0;
}
int test1()
{

  int i=0,j=0;
  i = j+ add(1,2);
  return 0;
}
