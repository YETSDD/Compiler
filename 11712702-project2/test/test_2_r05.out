Error type 8 at Line 2: incompatiable return type
Error type 5 at Line 8: unmatching type int and float
Error type 5 at Line 8: unmatching type int and error
